packages/features/ee/dsync/lib/handleGroupEvents.ts

```diff
@@ -178,28 +203,31 @@ const handleGroupEvents = async (event: DirectorySyncEvent, organizationId: numb
           isAutoJoin: true,
           currentUserParentTeamName: org.name,
           orgSlug: null,
-        });
-      }),
-      ...newOrgMembers.map((user) => {
-        return createAProfileForAnExistingUser({
-          user: {
-            id: user.id,
-            email: user.email,
-            currentUsername: user.username,
-          },
-          organizationId,
```

Comment:
For groups with 300 users, this was causing DB connection pool errors since we were making individual DB calls for each user.